src/dropdown/dropdown.js

```diff
@@ -90,6 +91,27 @@ angular.module('ui.bootstrap.dropdown', [])
       self.toggleElement[0].focus();
     }
   };
+  
+  scope.selectOption = function( evt ) {
+    var elems = angular.element(self.$element).find('.dropdown-menu a');
```

Comment:
It would be better to create a directive where when a value is true, then it focuses on the element.